src/components/DocumentReference/DocumentReferenceComponent.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import * as PropTypes from 'prop-types';
+import TableContainer from '@material-ui/core/TableContainer';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import DocumentReferenceComponentStyles from './DocumentReferenceComponent.style';
+import { formatDateString, getCodingDisplay } from '../common/HealthInfo/FhirResourcesUtils';
+
+const DocumentReferenceComponent = ({ data, consentReqId }) => {
+  const authorArray = [];
+
+  function extractAuthor(entry) {
+    if (entry.author) {
+      entry.author.forEach((author) => authorArray.push(author.display));
+      return authorArray;
+    }
+    return undefined;
+  }
```

Comment:
This can be simplified to 
```
function extractAuthor(entry) {
  if(!entry.author) return;
  return entry.author.map(author => author.display);
}
```

Reply:
done